x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/logs/logs_tab_content.tsx

```diff
@@ -23,6 +25,8 @@ export const LogsTabContent = () => {
   const { getDateRangeAsTimestamp } = useUnifiedSearchContext();
   const { from, to } = useMemo(() => getDateRangeAsTimestamp(), [getDateRangeAsTimestamp]);
   const { hostNodes, loading } = useHostsViewContext();
+  const { services } = useKibana<InfraClientStartDeps>();
```

Comment:
**nit:** prefer the usage of `useKibanaContextForPlugin`, it is already typed to include all the `infra` plugin deps.